### DISCUSSION TITLE
How to change layout in Calculations edit

### DISCUSSION THREAD

**Discussion by MaxPegankin** [YetiForceCompany/YetiForceCRM#14814] [How to] [answered]
Hi All, 
Who knows how to change edit layout in Calculations module? What file needs to be changed?
![image](https://user-images.githubusercontent.com/54612661/121477338-50d3d900-c9fa-11eb-83fe-6e855070dd1e.png)

**Comment by vovpff** [comment, marked as answer]
`/config/Performance.php`

```
	/** Is divided layout style on edit view in modules with products */
	public static $INVENTORY_EDIT_VIEW_LAYOUT = false;

	/** List of modules with splitted edit view layout */
	public static $MODULES_SPLITTED_EDIT_VIEW_LAYOUT = [];
```

**Comment by mariuszkrzaczkowski** [comment]
check this
https://github.com/YetiForceCompany/YetiForceCRM/blob/cee09e447d10f11355c235815c8bdcbe8d72e401/config/Performance.php#L131-L135

**Comment by rskrzypczak** [comment]
Currently, it is not possible to make such a change for only one module.
There are only global settings for all advanced modules.
as pointed out by vovpff :)

**Comment by garryatbrighton** [comment]
Somehow it not works with custom module (latest stable). Any tips?
